{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/services.py

```diff
@@ -0,0 +1,19 @@
+# Third Party Stuff
+from django.conf import settings
+from django.contrib.auth import get_user_model, authenticate
+
+# {{cookiecutter.main_module}} Stuff
```

Comment:
As per [this](https://github.com/Fueled/django-init/blob/master/%7B%7Bcookiecutter.github_repository%7D%7D/.editorconfig#L22), this should be `{{ cookiecutter.project_name }} Stuff`